esp/src/eclwatch/DFUQueryWidget.js

```diff
@@ -306,8 +316,8 @@ define([
                 arrayUtil.forEach(this.desprayGrid.store.data, function (item, idx) {
                     var request = domForm.toObject(context.id + "DesprayForm");
                     request.destPath = context.desprayTargetPath.getDropZoneFolder();
-                    if (!context.endsWith(request.destPath, "/")) {
-                        request.destPath += "/";
+                    if (!context.endsWith(request.destPath, request.destPath)) {
+                        request.destPath += request.destPath;
```

Comment:
What if a user types in a subfolder, ex. \subdir1? Does the request.destPath still return "\\".